components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -165,15 +165,19 @@ export const buildModel = ({
 		latestItemsTime,
 		latestItemsAgeHours
 	})
-
 	if (itemGroups.length > 0 && customSlotContent) {
-		const insertPosition = Math.min(customSlotPosition, items.length)
-		const insert = getGroupAndIndex(itemGroups, insertPosition)
-		const copyOfItems = [...itemGroups[insert.group].items]
+		customSlotContent = Array.isArray(customSlotContent) ? customSlotContent : [customSlotContent]
+		customSlotPosition = Array.isArray(customSlotPosition) ? customSlotPosition : [customSlotPosition]
+
+		customSlotContent.forEach((item, index) => {
+			const insertPosition = Math.min(customSlotPosition[index], items.length)
```

Comment:
Considering that the `items` array only includes the news articles, I wonder if the total items should be `items.length + customSlotPosition.length`?

e.g.,
```suggestion
			const insertPosition = Math.min(customSlotPosition[index], customSlotPosition.length + items.length)
```

Reply:
You're right, this was atrocious of me to miss.

Reply:
It would have to be `items.length + customSlotPosition.length - 1`, otherwise it would overflow the array.